src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/cas2/ApplicationNoteServiceTest.kt

```diff
@@ -39,6 +40,7 @@ import java.util.UUID
 
 class ApplicationNoteServiceTest {
   private val mockApplicationRepository = mockk<Cas2ApplicationRepository>()
+  private val mockAssessmentRepository = mockk<Cas2AssessmentRepository>()
```

Comment:
are there unit tests for the new service method? Quite minor, but could they be moved to the same commit that adds the method, so this is a 'complete'/atomic commit?

Reply:
Have squashed these commits into one now.